tools/scripts/ptformat

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
```

Comment:
I take no responsibility around this file.

Reply:
I like this files. It has solid functionality